packages/app-contracts/src/Codes/ValidateCode.tsx

```diff
@@ -70,7 +70,6 @@ class ValidateCode extends React.PureComponent<Props> {
 
 export default translate(
   withCalls<Props>(
-    ['query.contracts.codeStorage', { paramName: 'codeHash' }], // 2.x
-    ['query.contract.codeStorage', { paramName: 'codeHash' }] // 1.x
+    ['query.contracts.codeStorage', { fallback: 'query.contract.codeStorage', paramName: 'codeHash' }]
```

Comment:
I like this.

Reply:
Thanks! :) The missing method calls were throwing error messages that I couldn't get rid of so this seemed like the more practical way

Reply:
Yes, it is actually funny - originally it wasn't displayed, we added the logging of that quite recently (it certainly helps in finding exactly what goes wrong).